src/pages/SignIn/components/UserSignIn.jsx

```diff
@@ -20,19 +21,28 @@ const UserSignIn = () => {
 
   return (
     <div className="sign-in-container">
-      <h1>Sign In</h1>
       <form id="myForm" onSubmit={signIn}>
-        <input
-          type="email"
-          placeholder="Enter your Email"
-          value={email}
-          onChange={(e) => setEmail(e.target.value)}></input>
-        <input
-          type="password"
-          placeholder="Enter Your Password"
-          value={password}
-          onChange={(e) => setPassword(e.target.value)}></input>
-        <button type="submit">Log In</button>
+        <div className="flex w-[400px] mx-auto flex-col flex-wrap mb-6 md:mb-0 gap-4">
+          <Input
+            size="md"
+            type="email"
+            variant={'bordered'}
+            label="Email"
+            value={email}
```

Comment:
NextUI Input component already uses useState internally, so you don't need to do this. Look at the comments that I left in Tiana's PR(https://github.com/rainawan/BruinDictionary/pull/21/files)

Reply:
done. changed email and password states to useRef